### ISSUE TITLE
Scrum 2 Report

### ISSUE THREAD

**Issue by pukkuKim** [czhaoca/TheKimsPlusTwo#27]
What did you work on this past iteration (2 weeks)?
What were any major issues/challenges you ran into?
What do you plan to work on for this coming iteration (2 weeks)?
--INCLUDE YOUR CSID IN YOUR POST--
Other Notes:
These are individual, so you are tracking your OWN work and duties
Can use this as a resource when you’re writing about your project on your resume or LinkedIn -> look back and remember details about implementation
We are having this due AFTER each workshop so that you are better able to understand the technology you’re going to be using
The real effort comes from talking to your group and planning your next steps!
Keep it short, succinct, and summarize the key points

Total: ~5 sentences max!

See Grading Rubric here: https://blogs.ubc.ca/cpsc4552023s/assessment-rubrics/#sec-2

DUE: Sunday, June 11 at 10PM (PST)

**Comment by chohyeonkim** [comment]
csid : w7n0n

1. During the past iteration  I worked on the navigation bar and also took part in reviews on github. I additionally contributed to the documentation via Readme.md.

2. I have some challenges with the responsive design and faced difficulties in task delegation among team members.

3. I plan to continue improving the frontend, adding more options to the navigation bar, fix the bugs,  and potentially implementing an authentication page.

**Comment by czhaoca** [comment]
CSID: s6v0b

1. Over the past two weeks, I focused on my part of building PropertyCard and PropertyGrid components which render on the home page.

2. In this sprint, the lack of a dedicated resource/person to organize notes and generate user stories/tickets has led to confusion over details and uncertainty regarding team consensus on tickets. Without well-defined tickets, such as clear Definitions of Done or Acceptance Criteria, we may face ongoing challenges.

3. I plan to refactor my part to fit into the group's unified style, and work for mock dataset and join the team to make final decision on API usage.

*********************************
Kudos to @jieunkxx for your outstanding work in provisioning the complete repository and stepping up to assist us in organizing the project. Your efforts are greatly appreciated.

**Comment by chris718** [comment]
CSID: e3e4o

1. During the past iteration, I focused on developing the detailed page component for a house description page. I created a basic version of the page that includes essential information. This component will serve as a foundation for further enhancements.

2. One of the major challenges I encountered was my limited experience with React and Redux while aiming to develop a visually appealing webpage.

3. For the upcoming iteration, my plan is to focus on polishing the existing component I developed and work on the data design with the team members.

**Comment by jieunkxx** [comment]
csid: j4n4c

1. - Project infrastructure such as installing libraries & frameworks, setting up code formatters, and created frontend/backend code architectures
    - Property images carousel on property detailed page and integrated functions and stylings throughout the app
2. - The ambiguity of the project's direction due to the delay in decision-making on project data resources was the major issue; however, in future sprints, we will set clearer goals and timelines to improve performance and form a path for the team's success. 
3.  - Implement Redux actions for listing properties & "likes" functions, and interface with the MLS API

**Comment by pukkuKim** [comment]
CSID: b2j9q
1. I implemented one of the webpages, "Mypage", by putting together 3 different components (i.e. user profile, user information, and user's favourite properties) on it.
2. I struggled with the layout. The current version is somewhat similar to how I want the page to look, but it's lacking the details.
3. I will improve the implementation of the frontend side of the page I've been working on. Also, the team as a whole will make the decision on which API(s) we'll be using for our project and design the DB schema.
